src/views/v2/components/events-bar/views/list/item.php

```diff
@@ -13,17 +13,27 @@
  * @var string $public_view_slug Slug of the view currently being listed.
  * @var object $public_view_data Data of the view currently being listed.
  *
- * @version 6.12.0
  * @since 5.3.0
  * @since 6.12.0 Add aria-label to the view selector list item link.
+ * @since TBD Add aria-current to active list item.
+ *
+ * @version TBD
  */
 
 $list_item_classes = [ 'tribe-events-c-view-selector__list-item', "tribe-events-c-view-selector__list-item--$public_view_slug" ];
+$is_active         = false;
+
 if ( $view_slug === $public_view_slug ) {
 	$list_item_classes[] = 'tribe-events-c-view-selector__list-item--active';
+	$is_active           = true;
 }
 ?>
-<li class="<?php echo esc_attr( implode( ' ', $list_item_classes ) ); ?>">
+<li
+	class="<?php echo esc_attr( implode( ' ', $list_item_classes ) ); ?>"
+	<?php if ( $is_active ) : ?>
+		aria-current="true"
```

Comment:
This attribute should go on the link.
[See the MDN example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current#example)